login.js

```diff
@@ -1,23 +1,228 @@
 const emailInput = document.getElementById("formInputEmail");
 const passwordInput = document.getElementById("formInputPassword");
 const loginButton = document.getElementById("loginButton");
+const togglePw = document.querySelector(".form-field-passVis");
+const togglePwImg = document.getElementById("passVisImg");
+const errorBorder = document.getElementsByClassName("error-border");
+const errorText = document.getElementsByClassName("error-text");
+//error-border[0] - email
+//error-border[1] - pw
 
-function checkEmailPwValidity() {
-  if (emailInput.checkValidity() && passwordInput.checkValidity()) {
-    console.log("둘다유효");
+const modal = document.getElementById("loginModal");
+const modalCloseBtn = document.getElementById("modalCloseBtn");
+const modalMsg = modal.querySelector(".modal-message");
+
+//모달 띄우기
+function showModal(message) {
+  modalMsg.textContent = message;
+  modal.classList.remove("hidden");
+}
+//모달 닫기
+modalCloseBtn.addEventListener("click", () => {
+  modal.classList.add("hidden");
+  if (loginCondition) {
+    window.location.href = "index.html"; //추후 items.html로 이동
+  }
+});
+
+const USER_DATA = [
```

Comment:
```suggestion
const users = [
```

보통 UPPER_SNAKE_CASE의 경우 절대적인 상수에(const 아님) 많이 쓰입니다. 이 경우에 회원은 변경되는 데이터일 수 있기 때문에 camel case를 사용하는 것이 좋습니다.

추가로 data같은 postfix는 아무런 정보를 주지 않는 postfix이므로 지양하는것이 좋습니다.